core/templates/dev/head/pages/exploration_player/learnerContinueButton.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
```

Comment:
Sorry @satishboggarapu -- I didn't realize, but the name of this file should also be changed to ContinueButtonDirective.js.